pkg/agent/datamodel/const.go

```diff
@@ -134,15 +134,15 @@ const (
 
 const (
 	Nvidia470CudaDriverVersion = "cuda-470.82.01"
```

Comment:
Wanted to confirm 470 version doesn't need its prefix removed.

Reply:
We've stopped publishing the 470 version and it's only used for a much older SKU. So we're keeping it as it is for now.